lib/editor/util/timetable.js

```diff
@@ -40,12 +40,27 @@ export function getHeaderColumns (
   return columns.filter(c => c.type !== 'DEPARTURE_TIME')
 }
 
-export function parseTime (timeString: string) {
-  const date = moment().startOf('day').format('YYYY-MM-DD')
-  return moment(date + 'T' + timeString, TIMETABLE_FORMATS).diff(
-    date,
-    'seconds'
-  )
+/**
+* Handles pasted data from clipboard (e.g. from CSV file)
+* If departure/arrival time cell, pastes in time format, otherwise returns string as is
+ */
+let alertShown = false
+export function parseCellValue (timeString: string, col: TimetableColumn) {
+  if (isTimeFormat(col.type)) {
+    const date = moment().startOf('day').format('YYYY-MM-DD')
+    const parsedDate = moment(date + 'T' + timeString, TIMETABLE_FORMATS, true)
+    if (!parsedDate.isValid()) {
+      if (!alertShown) {
+        alert('Please enter a valid time format')
```

Comment:
It would be good to leave the contents of the cells in the case that an invalid time format was entered

Reply:
I am not observing this issue. I see the original cell value persist after entering an invalid time format then entering or clicking out of cell. Do you see the cells go blank? I might be missing edge cases that I am not aware of. lmk if it persists after you pull my latest changes